ansible/library/conn_graph_facts.py

```diff
@@ -79,18 +79,15 @@
               "HwSku": "Arista-7260QX-64",
               "Type": "FanoutLeaf"
             },
-          "device_conn": [
-          {
-             "StartPort": "Ethernet0",
-             "EndPort": "Ethernet33",
-             "StartDevice": "str-s6000-on-1",
-             "VlanID": "233",
-             "BandWidth": "40000",
-             "VlanMode": "Access",
-             "EndDevice": "str-7260-01"
-           },
-           {...}
-           ],
+          "device_conn": {
```

Comment:
Why change this part?

Reply:
Previous comment is outdated.

Reply:
I updated the comment according to [conn_graph_facts.py:L210](https://github.com/sonic-net/sonic-mgmt/blob/master/ansible/library/conn_graph_facts.py#L210)